sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getApplicationProgressDetails.e2e-spec.ts

```diff
@@ -120,6 +136,46 @@ describe("ApplicationStudentsController(e2e)-getApplicationProgressDetails", ()
         appealStatus: StudentAppealStatus.Pending,
         applicationOfferingChangeRequestStatus:
           ApplicationOfferingChangeRequestStatus.InProgressWithStudent,
+        assessmentTriggerType: application.currentAssessment.triggerType,
+      });
+  });
+
+  it("Should get application progress details when the current assessment trigger type is 'Related application changed'.", async () => {
+    // Arrange
+
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { student },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+        createSecondDisbursement: true,
+      },
+    );
+    const [firstDisbursement, secondDisbursement] =
+      application.currentAssessment.disbursementSchedules;
+
+    application.currentAssessment.triggerType =
+      AssessmentTriggerType.RelatedApplicationChanged;
```

Comment:
Suggestion and optional, we can supply initial values for currentAssessment, first and second disbursement in the factory. 

![image](https://github.com/bcgov/SIMS/assets/54600590/15963eaa-b8cb-4c46-b88d-713d61f193aa)